addons/website_event_track/models/event_track.py

```diff
@@ -386,10 +384,6 @@ def _read_group_stage_ids(self, stages, domain, order):
         """ Always display all stages """
         return stages.search([], order=order)
 
-    def _synchronize_with_stage(self, stage):
-        if stage.is_done:
-            self.is_published = True
```

Comment:
We don't want to automatically publish tracks anymore when they enter a certain stage?